src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/SdkLimitOptions.cs

```diff
@@ -38,17 +42,21 @@ public SdkLimitOptions()
 
     internal SdkLimitOptions(IConfiguration configuration)
     {
-        // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#attribute-limits
+        // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#attribute-limits
```

Comment:
Add the tag so it links to the actual version of the specification being followed. Same to the other comments

```suggestion
        // https://github.com/open-telemetry/opentelemetry-specification/blob/v1.25.0/specification/configuration/sdk-environment-variables.md#logrecord-limits
```

Reply:
6f2d89a88fd84ec0c9affe6e7121a196adaf6497